### PR TITLE
Fix for missing additional features fields when creating new popup windows

### PR DESCRIPTION
**Pull request by MichaelVasseur**
The non-standard features (like **this-is-not-a-standard-feature**) given to window.open() are lost in the popup window creation workflow. This leads to the 'new-window' event handler not receiving these 'additional' features. These features may be used by applications to give some specific data to the popup.

NOTE : This pull request depends on pull request [#234](https://github.com/electron/libchromiumcontent/pull/234) in libchromiumcontent repo.

### CODE REVIEW

**Comment by zeke** [comment]
@MichaelVasseur can you add a note to https://github.com/electron/electron/blob/master/docs/api/window-open.md explaining the change you've made here?

**Comment by MichaelVasseur** [comment]
@zeke I pushed the changes to the documentation. Thanks for your feedback !

File: atom/browser/api/atom_api_app.cc [hunk 1]
```diff
@@ -525,6 +525,7 @@ void App::OnLogin(LoginHandler* login_handler,
 void App::OnCreateWindow(const GURL& target_url,
                          const std::string& frame_name,
                          WindowOpenDisposition disposition,
+						 const std::vector<base::string16>& features,
```
Comment (MarshallOfSound) [inline atom/browser/api/atom_api_app.cc]: This indenting seems off by quite a bit, it should be aligned with the line above

File: atom/browser/api/atom_api_app.h [hunk 2]
```diff
@@ -50,6 +50,7 @@ class App : public AtomBrowserClient::Delegate,
   void OnCreateWindow(const GURL& target_url,
                       const std::string& frame_name,
                       WindowOpenDisposition disposition,
+					  const std::vector<base::string16>& features,
```
Comment (MarshallOfSound) [inline atom/browser/api/atom_api_app.h]: Indenting again, maybe your editor is configured from tabs / spaces incorrectly?

**Comment by MichaelVasseur** [comment]
@MarshallOfSound Indentation issues fixed.

**Comment by MichaelVasseur** [comment]
Remaining errors are dues to C++ interfaces changes required (introduced in #234) #

**Comment by zcbenz** [comment]
👍

**Comment by MichaelVasseur** [comment]
I added a new PR #7494 to unit test this change.

**Pull request merged**